analyzer/tests/unit/test_option_parser.py

```diff
@@ -373,6 +373,20 @@ def test_ccache_compiler(self):
         self.assertEqual(res,
                          '/usr/lib/ccache/g++')
 
+    def test_compiler_gcc_implicit_includes(self):
+        action = {
+            'file': 'main.cpp',
+            'directory': '',
+            'command': 'cpp main.cpp'
```

Comment:
should this be g++ instead of cpp (preprocessor)?
will this be true for all g++ (on the travis)?